### PR TITLE
Use new semconv artifact

### LINKED ISSUE #1444: Nightly build #765 failed
[Nightly build #765](https://github.com/signalfx/splunk-otel-java/actions/runs/6181337294) failed.
Please take a look and fix it ASAP.

### PR DESCRIPTION
**Pull request by mateuszrzeszutek**
Fixes #1444

### CODE REVIEW

**Comment by breedx-splk** [comment]
Hmmm, it's unhappy about this: 
```
OpenTelemetry Javaagent failed to start
java.lang.NoClassDefFoundError: io/opentelemetry/javaagent/shaded/io/opentelemetry/semconv/ResourceAttributes
```

File: profiler/build.gradle.kts [hunk 1]
```diff
@@ -26,7 +26,7 @@ dependencies {
   compileOnly("io.opentelemetry:opentelemetry-sdk-extension-autoconfigure-spi")
   compileOnly("io.opentelemetry.javaagent:opentelemetry-javaagent-extension-api")
   compileOnly("io.opentelemetry.instrumentation:opentelemetry-instrumentation-api")
-  compileOnly("io.opentelemetry:opentelemetry-semconv")
+  compileOnly("io.opentelemetry.semconv:opentelemetry-semconv")
```
[the remark this answers is not in the document]
Comment (mateuszrzeszutek) [reply profiler/build.gradle.kts:29]: >Are we expecting to have it shaded from upstream or ?

Yup, exactly that.

**Comment by mateuszrzeszutek** [comment]
> Hmmm, it's unhappy about this:
> 
> ```
> OpenTelemetry Javaagent failed to start
> java.lang.NoClassDefFoundError: io/opentelemetry/javaagent/shaded/io/opentelemetry/semconv/ResourceAttributes
> ```

Hmm, we might have to wait until the new module is used in the upstream. I'll make this PR a draft for the time being.

**Comment by mateuszrzeszutek** [comment]
It's ready for review now; it should fix https://github.com/signalfx/splunk-otel-java/issues/1444 as well

**Review by laurit — APPROVED** [review]

**Pull request merged**